src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+
+  for (const key in robot) {
+    if (newRobot[robot[key]]) {
```

Comment:
2.[checklist] use correct check for property presence in object. Some properties can be present, but still contain falsy value.
GOOD EXAMPLE:
if (key in robot) {
EVEN BETTER EXAMPLE:
if (robot.hasOwnProperty(key)) {
BAD EXAMPLE:
if (robot[key]) {